release-notes/aws/v10.1.2.md

```diff
@@ -0,0 +1,24 @@
+# :zap: Giant Swarm Release 10.1.2 for AWS is now active for you! :zap:
+
+This release fixes a problem with the chart-operator cluster role. The changes
+only affect managed apps and upgrading to this release will not cause tenant
+cluster nodes to be rolled.
```

Comment:
@cokiengchiara @puja108 Sorry I already wrote this draft. 😅 But please change it if it's wrong.

Chiara, I'd like to get this release shipped pretty quickly. Since customers can hit the bug in 10.1.1. There will be more time for the 9.0.1 release.

Reply:
Thanks Ross! I will look at this first thing in the morning - is that ok? I just want to make the value clearer to customer, like what problem they evaded b/c of this release...

Reply:
@cokiengchiara Yes sure that's fine.

This release is a bug fix for chart-operator. If it crashes then we can't install or update apps.

Reply:
It fixes chart-operator? But i see here https://github.com/giantswarm/releases/blob/eb1f497299d758eaad227dd6297c3af1cee94df7/aws.yaml

that from aws 10.1.2 to 10.1.2, i see cluster-operator changed from 2.0.1 to 2.0.2 but chart operator is the same at 0.7.0

Reply:
This release fixes a problem with the chart-operator cluster role, which is responsible for installing and updating Managed Apps. The changes affect only Managed Apps and upgrading to this release will not cause cluster nodes to be rolled. **To prevent encountering problems when installing or updating an app, please upgrade to this release.**

@pipo02mix WDYT about the above? esp. the last line?

Reply:
I think it is fine and clear

Reply:
Thanks changes made. PTAL